src/main/resources/migrations/common/V2025.05.15__public_private_comments.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE video_booking ADD COLUMN notes_for_staff VARCHAR(1000);
+
+ALTER TABLE video_booking ADD COLUMN notes_for_prisoners VARCHAR(1000);
+
+ALTER TABLE prison_appointment ADD COLUMN notes_for_prisoners VARCHAR(1000);
```

Comment:
Could the prison_appointments table just contain a single comments column? And the service only populates from comments or notesForPrisoners, depending on the feature being active? It would never put notesForStaff in there.

Reply:
I suppose it could.  I did this more from a consistency point of view 🤔

Reply:
I wonder if there is a risk we end up with private comments being copied over by mistake.  I'd feel happier if we had a clean separation.

Reply:
ok, valid reason for separation then. All good.